steampy/market.py

```diff
@@ -52,6 +52,20 @@ def get_my_market_listings(self) -> dict:
         listings = get_market_listings_from_html(response.text)
         listings = merge_items_with_descriptions_from_listing(listings, listing_id_to_assets_address,
                                                               assets_descriptions)
+        n_showing = int(text_between(response.text, '<span id="tabContentsMyActiveMarketListings_end">', '</span>'))
+        n_total = int(text_between(response.text, '<span id="tabContentsMyActiveMarketListings_total">', '</span>'))
+        if n_total > n_showing:
+            url = "%s/market/mylistings/render/?query=&start=%s&count=%s" % (SteamUrl.COMMUNITY_URL, n_showing, -1)
+            response = self._session.get(url)
```

Comment:
If there are more items than the ones shown I call the api to get all the items starting from n_showing.
count = -1 get all the items remaining